src/frontend/src/primitives/Button.tsx

```diff
@@ -1,8 +1,58 @@
 import {
   Button as RACButton,
   type ButtonProps as RACButtonsProps,
+  Link,
+  LinkProps,
 } from 'react-aria-components'
+import { cva, type RecipeVariantProps } from '@/styled-system/css'
 
-export const Button = (props: RACButtonsProps) => {
-  return <RACButton {...props} className="lk-button" />
+const button = cva({
+  base: {
+    display: 'inline-block',
+    paddingX: '1',
+    paddingY: '0.625',
+    transition: 'all 200ms',
+    borderRadius: 8,
+    cursor: 'pointer',
+  },
+  variants: {
+    variant: {
+      default: {
+        color: 'control.text',
+        backgroundColor: 'control',
+        '_ra-hover': {
+          backgroundColor: 'control.hover',
+        },
+        '_ra-pressed': {
+          backgroundColor: 'control.active',
+        },
+      },
+      primary: {
+        color: 'primary.text',
+        backgroundColor: 'primary',
+        '_ra-hover': {
+          backgroundColor: 'primary.hover',
+        },
+        '_ra-pressed': {
+          backgroundColor: 'primary.active',
+        },
+      },
+    },
+  },
+})
+
+type ButtonProps = RecipeVariantProps<typeof button> &
+  (RACButtonsProps | LinkProps)
+
+export const Button = (props: ButtonProps) => {
+  const [variantProps, componentProps] = button.splitVariantProps(props)
+  if ((props as LinkProps).href !== undefined) {
+    return <Link className={button(variantProps)} {...componentProps} />
+  }
+  return (
+    <RACButton
```

Comment:
nit-picking so sorryyy:

In panda.config.ts, I noticed that we're using `ra` as the abbreviation for the React Aria Component, but in this instance, we've used `RAC`. Could we standardize the abbreviation to avoid confusion?

Reply:
for sure